docs/src/enterprise/getstarted/quickstart.md

```diff
@@ -104,26 +104,28 @@ You can choose from the following options:
     ```
 
 === "Advanced (SSO Enabled)"
-    This setup uses OIDC as the SSO authentication method thus requiring a valid OIDC configuration.
+This setup uses OIDC as the SSO authentication method thus requiring a valid OIDC configuration.
 
     1. Create a `docker-compose.yaml` with the content below.
     2. Create a `.env` file with the configurations below in the same directory as the `docker-compose.yaml`, docker compose will automatically use that.
     3. Run `docker compose up` in the same directory as the `docker-compose.yaml` file.
     4. Validate the OIDC configuration:
-      * In your browser, go to <http://localhost:8080> to access lakeFS UI
+      * In your browser, go to <http://localhost:8000> to access lakeFS UI
       * Complete the Setup process, and login with your Admin credentials
       * Logout and try to login again, you will be redirected to the OIDC login page.
 
     `.env`
```

Comment:
All the `.env` formatting is broken. 
PLEASE STOP THIS AUTO FORMATTING OR WHATEVER YOUR IDE DOES.